apps/files_trashbin/tests/Sabre/TrashbinPluginTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\Files_Trashbin\Tests\Sabre;
+
+use OC\Files\FileInfo;
+use OC\Files\View;
+use OCA\Files_Trashbin\Sabre\ITrash;
+use OCA\Files_Trashbin\Sabre\RestoreFolder;
+use OCA\Files_Trashbin\Sabre\TrashbinPlugin;
+use OCA\Files_Trashbin\Trash\ITrashItem;
+use OCP\IPreview;
+use Sabre\DAV\Server;
+use Sabre\DAV\Tree;
+use Test\TestCase;
+
+class TrashbinPluginTest extends TestCase {
+	private Server $server;
+
+	protected function setUp(): void {
+		parent::setUp();
+
+		$tree = $this->createMock(Tree::class);
+		$this->server = new Server($tree);
+	}
+
+	/**
+	 * @dataProvider quotaProvider
+	 */
+	public function testQuota(int $quota, int $fileSize, bool $expectedResult): void {
+		$fileInfo = $this->createMock(ITrashItem::class);
+		$fileInfo->method('getSize')->willReturn($fileSize);
+		$trashNode = $this->createMock(ITrash::class);
+		$trashNode->method('getFileInfo')->willReturn($fileInfo);
+		$restoreNode = $this->createMock(RestoreFolder::class);
+		$this->server->tree->method('getNodeForPath')->willReturn($trashNode, $restoreNode);
+		$previewManager = $this->createMock(IPreview::class);
+		$view = $this->createMock(View::class);
+		$view->method('free_space')->willReturn($freeSpace);
+		$plugin = new TrashbinPlugin($previewManager, $view);
+		$plugin->initialize($this->server);
```

Comment:
```suggestion
		$fileInfo = $this->createMock(ITrashItem::class);
		$fileInfo->method('getSize')->willReturn($fileSize);

		$trashNode = $this->createMock(ITrash::class);
		$trashNode->method('getFileInfo')->willReturn($fileInfo);

		$restoreNode = $this->createMock(RestoreFolder::class);

		$this->server->tree->method('getNodeForPath')->willReturn($trashNode, $restoreNode);

		$previewManager = $this->createMock(IPreview::class);

		$view = $this->createMock(View::class);
		$view->method('free_space')->willReturn($freeSpace);

		$plugin = new TrashbinPlugin($previewManager, $view);
		$plugin->initialize($this->server);
```
Just add some empty lines to make it more readable.